docs/testing.adoc

```diff
@@ -29,6 +29,20 @@ $ npm run test:func
 npm run test:funcsmoke
 ----
 
+Functional Tests support following (optional) variables:
+
+1. `DIRECT_CONNECT` (default=false) - When DIRECT_CONNECT is set, protractor will not use selenuim and directly connect to Chrome. Use this flag to run the tests faster.
+eg: `DIRECT_CONNECT=true npm run test:funcsmoke`
+
```

Comment:
@jarifibrahim please change this numbered list to bulleted list since these are not sequential steps.

When DIRECT_CONNECT>When `DIRECT_CONNECT`

line 35, 40, and 44 - eg:> For example,

line 34 and 37 - Protractor - capital P